Gordon360/Models/CCT/Housing/Year.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <auto-generated> This file has been auto generated by EF Core Power Tools. </auto-generated>
+#nullable disable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Gordon360.Models.CCT;
+
+[PrimaryKey("ApplicationID", "Year1")]
+[Table("Year", Schema = "Housing")]
+public partial class Year
```

Comment:
What is the purpose of this table? If it is a 1:1 relationship with `Application` and has only 1 added field, why is it not just a column on the `Application` table?